lib/renderer/webview/webview.h

```diff
@@ -2213,12 +2239,16 @@ struct webview_priv
   WEBVIEW_API void webview_dialog(struct webview *w,
                                   enum webview_dialog_type dlgtype, int flags,
                                   const char *title, const char *arg,
-                                  char *result, size_t resultsz)
+                                  char *result, size_t resultsz, char *filter)
   {
     if (dlgtype == WEBVIEW_DIALOG_TYPE_OPEN ||
         dlgtype == WEBVIEW_DIALOG_TYPE_SAVE)
     {
       NSSavePanel *panel;
+      NSString *filter_str = [NSString stringWithUTF8String:filter];
+      filter_str = [filter_str stringByReplacingOccurrencesOfString:@"*."
+                                     withString:@""];
+      NSArray *fileTypes = [filter_str componentsSeparatedByString:@","];
```

Comment:
Do we need to trim spaces? Will it accept `*.mpg, *.mpeg `?

Reply:
Good point will add trimming in Goland since I think it should be applied to all platforms